doc/nrf/ug_matter_overview_network_topologies.rst

```diff
@@ -56,10 +62,14 @@ Edge Router
 Fabric
   This is a logical set of nodes that communicate with each other and can belong to different networks.
   The devices in a fabric share the same :ref:`root of trust <ug_matter_network_topologies_concepts_security>` and configuration state and are identifiable with a unique 64-bit Fabric ID.
-  Using the Multiple Fabrics feature, you can assign nodes to a specific set of fabrics.
+
+Multi-fabric
+  A Matter feature that allows to commission a Matter node to different fabrics (each of which has an administrator of its own) and to communicate with other devices within these fabrics.
 
 Multi-admin
-  A Matter feature that allows to connect Matter devices to multiple applications and ecosystems out of the box.
```

Comment:
I think that "Multi-admin" and "Multi-fabric" are the same thing (see https://github.com/CHIP-Specifications/connectedhomeip-spec/commit/2f06507cba128f5c43f8fd32f5402ab5bb53a4c3#diff-98348510935f021ef63e60b31389b5a3e54cb3e6f679298e7c57e078efa68e60 commit that replaced the former with the latter). The names just emphasize different aspects of it, but I would treat them as synonyms ("Multi-fabric" being more accurate though).